OWNERS_ALIASES

```diff
@@ -11,4 +11,3 @@ aliases:
   - steuhs
   - peterfeifanchen
   - efiturri
-  - nbarthwal
```

Comment:
This PR just removes Nikhil from the reviewers, but still not all oncalls will have the `approve` power, so it's different from the PR description?

Reply:
`git commit -am` :(

Thanks for noting